lib/Minz/FrontController.php

```diff
@@ -105,10 +105,7 @@ public function run () {
 	* Permet d'arrêter le programme en urgence
 	*/
 	private function killApp ($txt = '') {
-		if ($txt == '') {
-			$txt = 'See logs files';
-		}
-		exit ('### Application problem ###<br />'."\n".$txt);
+		exit (errorMessage ('Application problem', $txt) );
```

Comment:
We should not call from Minz a function that does not exist there. I will make a suggestion

Reply:
I think I dealt with everything else, but not sure how to fix this problem.

Reply:
Let me have a look